pkg/templates/crds/grc/OWNERS

```diff
@@ -0,0 +1,16 @@
+approvers:
```

Comment:
Do we need to set up similar owner files for the other CRD directories or is this GRC specific?

Reply:
This came from a request by GRC in https://github.com/stolostron/multiclusterhub-operator/pull/892. We can add more owners files upon request